From 3cf91d8d674a496d533aa106723a632b68ab26ba Mon Sep 17 00:00:00 2001
From: Tom Anderson <thomasanderson@chromium.org>
Date: Tue, 12 Dec 2017 23:31:56 +0000
Subject: [PATCH] Fix use_cups=false build

BUG=793801
R=thestig@chromium.org

Change-Id: I60ee6ec48d2686d0b42ca25311a9b76009ce9768
Reviewed-on: https://chromium-review.googlesource.com/823245
Reviewed-by: Lei Zhang <thestig@chromium.org>
Commit-Queue: Thomas Anderson <thomasanderson@chromium.org>
Cr-Commit-Position: refs/heads/master@{#523602}
---
 chrome/service/cloud_print/print_system.cc | 9 ++++++++-
 1 file changed, 8 insertions(+), 1 deletion(-)

--- a/chrome/service/cloud_print/print_system.cc
+++ b/chrome/service/cloud_print/print_system.cc
@@ -5,6 +5,7 @@
 #include "chrome/service/cloud_print/print_system.h"
 
 #include "base/guid.h"
+#include "build/build_config.h"
 
 namespace cloud_print {
 
@@ -35,5 +36,11 @@ std::string PrintSystem::GenerateProxyId
   return base::GenerateGUID();
 }
 
-}  // namespace cloud_print
+#if defined(OS_LINUX) && !defined(USE_CUPS)
+scoped_refptr<PrintSystem> PrintSystem::CreateInstance(
+    const base::DictionaryValue*) {
+  return nullptr;
+}
+#endif
 
+}  // namespace cloud_print
